src/SUFST/src/can_unpack.c

```diff
@@ -151,5 +173,28 @@ void queue_receive_thread_entry(ULONG input)
         {
           return;
         }
+        /* For statistic */
+        unpack_ptr->stats.tx_pdu_count++;
+        unpack_ptr->stats.txbits += (sizeof(pdu_struct) * 8);     
```

Comment:
Why is it `* 8`?
Whatever reason also, the `8` should be some kind of `#define` constant in this file.

Reply:
This is number of bits, and the sizeof gives the number of bytes, so that's simple conversion. We might use bytes per second, instead of bits per second if you wish.

Reply:
Ah okay yeah of course! Maybe count the number of bytes, then multiply it by 8 in the callback calculation. Less multiplications (not that it makes a huge difference). The same thing should also apply to the RTCAN messages.